### ISSUE TITLE
Sorting of wordforms is buggy

### ISSUE THREAD

**Issue by johnjcamilleri** [MLRS/gabra-api#6]
Output of `lexemes/wordforms/5200a366e36f237975000f26`:

```json
[
{
  "_id": "52089f05e36f230d56229803",
  "aspect": "perf",
  "generated": true,
  "lexeme_id": "5200a366e36f237975000f26",
  "phonetic": "kɪdbɛt",
  "polarity": "pos",
  "sources": [
    "Camilleri2013"
  ],
  "subject": {
    "person": "p3",
    "number": "sg",
    "gender": "f"
  },
  "surface_form": "kitbet"
},
{
  "_id": "52089f05e36f230d56229800",
  "aspect": "perf",
  "dir_obj": null,
  "generated": true,
  "ind_obj": null,
  "lexeme_id": "5200a366e36f237975000f26",
  "phonetic": "ktɪpt",
  "polarity": "pos",
  "sources": [
    "Camilleri2013"
  ],
  "subject": {
    "person": "p1",
    "number": "sg"
  },
  "surface_form": "ktibt"
},
```

Order here is clearly wrong.

**Comment by johnjcamilleri** [comment]
Added test in 23eb65a.
Something to do with null fields vs missing ones (`dir_obj` and `ind_obj`)

**Comment by johnjcamilleri** [comment]
Fixed in e68f193.
Missing fields now treated equally as `null` and `''`.
